sql/src/main/java/io/druid/sql/avatica/DruidMeta.java

```diff
@@ -91,9 +112,9 @@ public void openConnection(final ConnectionHandle ch, final Map<String, String>
     // Build connection context.
     final ImmutableMap.Builder<String, Object> context = ImmutableMap.builder();
     for (Map.Entry<String, String> entry : info.entrySet()) {
-      if (!SKIP_PROPERTIES.contains(entry.getKey())) {
+      //if (!SKIP_PROPERTIES.contains(entry.getKey())) {
```

Comment:
This was correct, I just forgot to remove the commented out lines and SKIP_PROPERTIES